src/services/licenseCodes.service.ts

```diff
@@ -186,7 +186,7 @@ export class LicenseCodesService {
           await this.tiersService.insertTierToUser(userId, tierProduct.id);
         }
       } else {
-        await this.storageService.changeStorage(user.uuid, maxSpaceBytes);
+        await this.storageService.updateUserStorageAndTier(user.uuid, maxSpaceBytes, '');
```

Comment:
Why an empty tier?

Reply:
Finally, I used the free tier for products that do not have any associated tier.